go/libraries/doltcore/doltdb/foreign_key_coll.go

```diff
@@ -61,18 +61,41 @@ func (f *ForeignKeyViolationError) Error() string {
 		}
 		key, _ := f.ViolationRows[i].NomsMapKey(f.Schema).Value(context.Background())
 		val, _ := f.ViolationRows[i].NomsMapValue(f.Schema).Value(context.Background())
-		valSlice, _ := val.(types.Tuple).AsSlice()
-		all, _ := key.(types.Tuple).Append(valSlice...)
-		str, _ := types.EncodedValue(context.Background(), all)
+
+		str := getRowValuesAsString(key, val)
 		sb.WriteRune('\n')
 		sb.WriteString(str)
 	}
+
+	var refCols []string
```

Comment:
These aren't referenced columns, these are just standard columns as you're using the `TableColumns` field. Referenced columns would be the `ReferencedTableColumns` field.